### DISCUSSION TITLE
What is damping in the LM method?

### DISCUSSION THREAD

**Discussion by xphsean12** [facebookresearch/theseus#462] [Q&A] [answered]
## ❓ Questions and Help
I raised this question because damping=0.1 and damping = 0 will result in much difference in the optimization result. Where could I find some references?

**Comment by luisenp** [comment, marked as answer]
Page 258 of [this](https://www.csie.ntu.edu.tw/~r97002/temp/num_optimization.pdf) book would be a good starting point. 

Also, our LM implementation is based on Section 4.1 of [this](https://people.duke.edu/~hpgavin/ce281/lm.pdf), method 1.

Let me know if you have further questions.
